Queryable/Queryable/View/LimitedLibraryPicker.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  LimitedLibraryPicker.swift
+//  Queryable
+//
+//  Created by Jaden Geller on 10/30/23.
+//
+
+import SwiftUI
+import PhotosUI
+
+struct LimitedLibraryPicker: UIViewControllerRepresentable {
+    @Binding var isPresented: Bool
+    
+    func makeUIViewController(context: Context) -> UIViewController {
+        .init()
+    }
+    
+    class Coordinator {
+        var isPresented = false
```

Comment:
Coordinator tracks whether or not the picker is _actually_ presented. Binding tracks whether or not the view wants it to be presented.

This allows us to: (a) prevent attempting to present the picker again when it is already presented, and (b) update the binding value back to `true` if the picker is still presented (since we cannot programmatically dismiss).

Technically, neither of these code paths are exercised by the app, but it seemed worth implementing this properly in case the calling code changes.